docker-compose.yml

```diff
@@ -0,0 +1,34 @@
+version: "2"
+services:
+  app:
+    build: .
+    links:
+      - db
+    volumes:
+      - .:/app
+    environment:
+      - DATABASE_URL=postgres://appuser:test123@db/test_project
+      - DJANGO_SETTINGS_MODULE=config.settings_interactive
+      - PYTHONUNBUFFERED=yup
+    working_dir: /app
+    entrypoint: ["./test_project/wait-for-postgres.sh"]
```

Comment:
Waiting for Postgres in the `entrypoint` is a _little_ overboard since not all operations on this container will necessarily require it (e.g. running `black`) but it feels like the easiest thing to do, and doesn't seem to take too much extra time.

Reply:
Works for me!